app/views/user.js

```diff
@@ -146,11 +182,20 @@ const User = {
 			);
 			console.log(result);
 			if (result.success) {
-				const index = this.profile.following_ids.indexOf(Number(this.id));
-				if (index > -1) {
-					let newFollowingIds = this.profile.following_ids;
-					newFollowingIds.splice(index, 1);
-					this.$set(this.profile, "following_ids", newFollowingIds);
+                if (userData.followingIds === undefined) {
+                    userData.followingIds = await this.getFollowings();
+                }
+                var filteredIds = userData.followingIds.filter(
+                        (value, index) => value !== this.id);
+                userData.followingIds = filteredIds;
+                store.set('userData', userData);
+                this.$set(
+                        this.user,
+                        'user_profile',
+                        {
+                            ...this.user.user_profile,
+                            notification_type:null,
+                            my_following: false});
 				}
 			} else {
```

Comment:
I just recalled before I tested this PR, I fixed a syntax error in here. There is a curly brace that is missplaced. I will correct it as soon as I can

Reply:
done